test/astarte_core/triggers/simple_triggers_protobuf_test.exs

```diff
@@ -0,0 +1,164 @@
+defmodule Astarte.Core.Triggers.SimpleTriggersProtobufTest do
+  use ExUnit.Case
+
```

Comment:
Add a describe block
```
describe "payload serialized with ExProtobuf"
```
so tests read naturally: "payload serialized with ExProtobuf still works for AMQPTriggerTarget" etc.